### ISSUE TITLE
Variable Named "of" Sometiems Confuses the Parser and Results in an Error

### ISSUE THREAD

**Issue by jpbro** [WaynePhillipsEA/twinbasic#752]
**Describe the bug**
Having a variable name "of" sometimes results in an "[Of] is unexpected here" error.

**To Reproduce**
Try the following code:

```
    Public Sub Main()
        Dim x As String
        Dim y As String
        Dim of As Long
        
        y = Left$(X, Of + 1)
    End Sub
```

**Expected behavior**
"Of" should be evaluated as a variable when used as a variable name.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC compiler version 0.14.31

**Additional context**
A variable named "of" works in some causes, like `of = of + 1`, but when passed to methods like Left$, Right$, Mid$, etc... it fails.

**Comment by jpbro** [comment]
There's no rush on these issues I'm posting today. I had a bit of time to try to get some of my DLLs running in tB and thought should post the results. I'm down from thousands of "Problems" in the list to just dozen or so now, so the progress is impressive! Thanks for all the hard work :)

**Comment by WaynePhillipsEA** [comment]
Thanks for your help in tracking them down!  I'll definitely get them fixed within a few days.  Some of them are quite _edgy_, but that's to be expected :)

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.14.32, thanks for reporting!

**Comment by WaynePhillipsEA** [comment]
> It seems new keywords clash comes up as significant part of backcompat issues that crop up on VBx code migration. Wish TB parser would allow and is prepared to handle proper all language keywords as identifiers (vars, params, function or method names, enums, types, module and class names too)

tB is already context-aware, and due to this is already generally much more forgiving than VBx.  There are of course going to be odd issues that crop up due to the weird hard-coded syntax exceptions baked into the VBx parser.  And ther'll also be odd ones like this that are self-inflicted :)
